src/app/shared/page-header/page-header.component.html

```diff
@@ -0,0 +1,84 @@
+<div class="affix-container">
```

Comment:
This has been removed from the seed project as its not doing anything anyway. There is a fix in for 1.1.0 to remove the need for wrapping this component in future.